Source/NSPopover.m

```diff
@@ -249,10 +249,12 @@ - (void) showRelativeToRect: (NSRect)positioningRect
   NSRect screenRect;
   NSRect windowFrame;
   NSRect viewFrame;
-
+  NSWindow *window = nil;
+  
   [_contentViewController loadView];
   view = [_contentViewController view];
-  viewFrame = [view frame];
+  window = [view window];
```

Comment:
I don't see where this new variable gets used.

Reply:
Right now things are in flux a bit.   I will be moving a lot of things to NSDebugLog or removing them once I am done.